torch/_dynamo/variables/torch.py

```diff
@@ -575,7 +575,7 @@ def check_any_unspec(x):
                     # NB: This includes UnspecializedPythonVariable
                     if isinstance(x, (TensorVariable, SymNodeVariable)):
                         return True
-                    elif isinstance(x, ListVariable):
+                    elif isinstance(x, (ListVariable, TupleVariable)):
```

Comment:
the dynamo pattern here is unpack_var_sequence check and access, should work for all collections.